app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="top-navbar-container">
+  <div id="top-navbar">
+    <ul>
+      <li><a href="https://library.stanford.edu/myaccount">My Account</a></li>
+      <li><a href='#'>Feedback</a></li>
```

Comment:
@jvine is there somewhere we should send this link in the interim?

Reply:
@jkeck you mean with no feedback form hooked up? there's always this:
http://library.stanford.edu/ask/email/feedback
but probably moot by now.